google/cloud/storage/internal/curl_client.cc

```diff
@@ -510,9 +510,9 @@ StatusOr<ObjectMetadata> CurlClient::InsertObjectMedia(
   }
 
   // If the application has set an explicit hash value we need to use multipart
-  // uploads.
-  if (!request.HasOption<DisableMD5Hash>() &&
-      !request.HasOption<DisableCrc32cChecksum>()) {
+  // uploads. We check `DisableCrc32cChecksum` only because `DisableMD5Hash`
+  // always has a value.
+  if (!request.HasOption<DisableCrc32cChecksum>()) {
     return InsertObjectMediaMultipart(request);
   }
```

Comment:
I would like to ask something here. When `DisableMD5Hash` is enabled by default, what is the condition to use `InsertObjectMediaMultipart` ?

Reply:
The CI will failed because of this condition. So please do not run the CI now. Thank you